docs/posts/into-snowflake/index.qmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Snow IO: loading data from other DBs into Snowflake"
+author: "Phillip Cloud"
+error: false
+date: "2024-02-23"
```

Comment:
can you update the date to today, then I'll merge? I don't have snowflake setup or I'd do it myself

Reply:
Done